app/Mail/Attendance/Report.php

```diff
@@ -0,0 +1,47 @@
+<?php
+
+declare(strict_types=1);
+
+namespace App\Mail\Attendance;
+
+use App\AttendanceExport;
+use Illuminate\Bus\Queueable;
+use Illuminate\Mail\Mailable;
+use Illuminate\Queue\SerializesModels;
+use Swift_Mime_SimpleMimeEntity as SimpleMimeEntity;
+
+class Report extends Mailable
+{
+    use Queueable;
+    use SerializesModels;
+
+    /**
+     * The AttendanceExport for the email.
+     *
+     * @var AttendanceExport
+     */
+    public $export;
+
+    /**
+     * Create a new message instance.
+     */
+    public function __construct(AttendanceExport $export)
+    {
+        $this->export = $export;
+    }
+
+    /**
+     * Build the message.
+     *
+     * @return $this
+     */
+    public function build()
+    {
+        return $this
+            ->from('attendancereports@my.robojackets.org', 'RoboJackets')
```

Comment:
Why this email?

Reply:
Usually we do noreply@my.robojackets.org and I didn't want that to appear when I'm telling people to reply to an email (there's a reply-to set). That particular address is arbitrary.